client/state/stats/lists/utils.js

```diff
@@ -974,19 +974,35 @@ export const normalizers = {
 		const emailsData = get( data, [ 'posts' ], [] );
 
 		return emailsData.map(
-			( { id, href, date, title, type, opens, clicks, opens_rate, clicks_rate } ) => {
+			( {
+				id,
+				href,
+				date,
+				title,
+				type,
+				opens,
+				clicks,
+				opens_rate,
+				clicks_rate,
+				opens_unique,
+				clicks_unique,
+				total_sends,
+			} ) => {
 				const detailPage = site ? `/stats/email/opens/day/${ id }/${ site.slug }` : null;
 				return {
 					id,
 					href,
 					date,
 					label: title,
 					type,
-					value: clicks || '0',
+					value: clicks_rate || '0',
 					opens: opens || '0',
 					clicks: clicks || '0',
 					opens_rate: opens_rate || '0',
 					clicks_rate: clicks_rate || '0',
+					opens_unique: opens_unique || '0',
+					clicks_unique: clicks_unique || '0',
```

Comment:
Was this coming from the backend as such? In the backend PR we have `unique_clicks` and here `clicks_unique`.

Internal ref 169248-ghe-Automattic/wpcom

Reply:
it comes as `clicks_unique` the api doc in 169248-ghe-Automattic/wpcom was wrong, fixed! good catch 🎄